docs/core-concepts/bot-sequences/actions.mdx

```diff
@@ -1,9 +1,659 @@
+sidebar_label: 'Bot Actions'
 ---
 
-# Actions
+# Bot Actions
 
-import Placeholder from '../../partials/_placeholder.mdx';
+Bot Actions are the core pieces of logic that drive behavior within a segment of your test sequence. Actions
+can range from simple operations like clicking or keypresses, to more complex operations like clicking on
+an object in the scene using computer vision.
 
-<Placeholder />
+## Structure of a Bot Action
+
+Bot Actions are defined in Bot Segments within the `botAction` field. A segment may only contain one bot action.
+
+```json
+// Segment structure
+{
+    "name": "Click the play button",
+    "description": "Clicks the play button in the main menu",
+    "endCriteria": [...],
+    "botAction": {
+        "type": ..., // The type of action to be taken, as a string
+        "data": {
+            ... // The data for the action, as a JSON object
+        }
+    }
+}
+```
+A bot action has a `type` and a `data` field. The `type` indicates the type of action to be taken, and the `data` field contains
+the parameters for that action. For example, a bot action to click on a coordinate may look like this:
+
+```json
+{
+    "name": "Click the play button",
+    "description": "Clicks the play button in the main menu",
+    "endCriteria": [...],
+    "botAction": {
+        "type": "InputPlayback",
+        "data": {
+            "startTime": 0.0,
+            "inputData": {
+                "keyboard": [],
+                "mouse": [
+                    {
+                        "apiVersion": 1,
+                        "startTime": 0.0, // This is the time offset at which the action should be performed
+                        "screenSize": {"x": 1920, "y": 1080},
+                        "position": {"x": 907, "y": 796}, // This is the location to click
+                        "worldPosition": null,
+                        "leftButton": true, // This indicates that the left mouse button should be pressed
+                        "middleButton": false,
+                        "rightButton": false,
+                        "forwardButton": false,
+                        "backButton": false,
+                        "scroll": {"x": 0, "y": 0},
+                        "clickedObjectNormalizedPaths": []
+                    }
+                ]
+            }
+        }
+    }
+}
+```
+
+:::info
+There are additional fields that can be set on a bot action and many of these `inputData` fields, such as an apiVersion. You can see the full
+specification in the [source code](https://github.com/Regression-Games/RGUnityBots/blob/cd68ed0f4b10567eb652477717338d92b4f4571e/src/gg.regression.unity.bots/Runtime/Scripts/StateRecorder/BotSegments/Models/BotAction.cs).
+:::
+
+## Action Types
+
+There are many different types of actions that can be used in a bot segment, each useful in their own way.
+The list of supported types are as follows:
+
+* `InputPlayback`: This action plays back a sequence of mouse and keyboard inputs.
```

Comment:
```suggestion
* `InputPlayback`: This action plays back a set of mouse and keyboard inputs.
```

Reply:
Updated, changed to "series" though to at least give the sense that ordering does matter (in the case where the start times are the same)